frontend/src/components/LoginPage.js

```diff
@@ -25,11 +29,15 @@ const LoginPage = (event) => {
         event.preventDefault();
         const formData = new FormData(event.target);
         const formDataObj = Object.fromEntries(formData.entries());
-        await userService.login(formDataObj);
-        navigate('/home')
+        const res = await userService.login(formDataObj)
+            .catch((err) => {
+                setAlertMessage(err.response.data.message);
```

Comment:
need to setAlertmessage(null) after a setTimeout function

Reply:
Done